src/main/java/com/openshift/jenkins/plugins/pipeline/Auth.java

```diff
@@ -320,7 +335,7 @@ public static String deriveBearerToken(String at, TaskListener listener,
      * e.printStackTrace(listener.getLogger()); } } return deriveBearerToken(at,
      * listener, verbose, vars, env); }
      */
-    public static String deriveCA(String ca, TaskListener listener,
+    /*public static String deriveCA(String ca, TaskListener listener,
```

Comment:
the method is at the moment not getting called .... but I decided to leave it in as a reference just in case it might be needed at some point

but i don't feel strongly about it, and can just delete if you prefer